### DISCUSSION TITLE
Hot to enable parsing of Escape Codes

### DISCUSSION THREAD

**Discussion by DelphiCreator** [landrix/Zint-Barcode-Generator-for-Delphi#30] [Q&A]
Hi,

fantastic work. But I cannot find out how to enable parsing of Escape Codes.
In Zint Studio there is a checkbox to enable it.
What am I overlooking?

Thanks Uli

**Comment by landrix** [comment]
Just concat the string you want.

For instance:

\E      0x04    EOT     End of Transmission

var str : String := '123a' + #04 + '345';

**Reply by DelphiCreator** [reply]
Works fine. Thank you very much.
